tools/pipelines/build-bundle-size-tools.yml

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# build-bundle-size-tools pipeline
+
+name: $(Build.BuildId)
+
+parameters:
+- name: releaseBuildOverride
+  displayName: Release Build (default = not released)
+  type: string
+  default: none
+  values:
+    - none
+    - prerelease
+    - release
+- name: publishOverride
+  displayName: Publish Override (default = based on branch)
+  type: string
+  default: default
+  values:
+    - default
+    - skip
+    - force
```

Comment:
These were copied from `build-client.yml` right? Do these apply the same way here? I'm wondering if `releaseBuildOverride` makes sense or not here since it's an internal package...?

Reply:
I copied it from build-test-tools.yml which is an internal package.  Curtis created that so I'm hoping it's correct enough 🙃